src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -146,7 +146,7 @@ public static bool ShowContainerPickerWindow(IntPtr hwnd, out string connectionS
             bool? dialogResult = dialog.ShowModal();
             if (dialogResult.GetValueOrDefault(false))
```

Comment:
>dialogResult [](start = 16, length = 12)

nit: I find if (dialogResult == true) a lot easier to read.

Reply:
won't fix